index.html

```diff
@@ -249,14 +249,34 @@ <h1>The Registration Process</h1>
 Any method name SHOULD avoid generic terms such as "mymethod" or "registry".
       </li>
       <li>
-If there are copyright, trademark, or any intellectual property rights
-concerns, the addition and use MUST be authorized in writing by the intellectual
-property rights holder under a
-<a href="https://en.wikipedia.org/wiki/Reasonable_and_non-discriminatory_licensing">F/RAND</a>
-license. Examples include DID Methods that use trademarked brand names,
-property names that utilize the titles of copyrighted works, and patented
-technology that would cause the use of the extension to require licensing a
-patent.
+Any DID Method submission MUST not impose any copyright, trademark, or 
+intellectual property rights concerns on the W3C, DID Method implementers, or 
+users.  To that end, the following apply:
+          <ol>
+          <li>
+DID Method submitters affirm to the best of their knowledge that their 
+submission will not infringe upon or otherwise violate the rights of any third 
+party.
+          </li>
+          <li>
+If there are any known copyright, trademark, or intellectual property rights 
+concerns, the addition and use of a DID Method MUST be authorized in writing 
+by the intellectual property rights holder under a 
+<a href="https://en.wikipedia.org/wiki/Reasonable_and_non-discriminatory_licensing">F/RAND</a> 
+license.
+          </li>
+          <li>
+If a DID Method is subsequently alleged to violate any copyright, trademark, 
+or intellectual property rights, those making the allegation SHALL provide a 
+copy of their granted legal rights documentation or an order from a court or 
+arbitral tribunal, in each case of competent jurisdiction.
+          </li>
+          <li>
+Should legally valid and enforceable copyright, trademark, or intellectual 
+property rights documentation be presented then, at the discretion of the 
+rights holder, the subject DID Method SHALL be removed.
+          </li>
+          </ol>
```

Comment:
```suggestion
        </ol>
```

Reply:
Thanks for the suggestions.  I incorporated them.